src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/signup/AlmostTherePageObject.java

```diff
@@ -31,20 +31,31 @@ public void verifyAlmostTherePage() {
     waitForElementByElement(changeMyEmail);
   }
 
-  private String getActivationLinkFromMail(String email, String password) {
+  private String getActivationLinkFromMail(String email, String password, String language) {
+    String mailSubject;
+    if (language == "ja") {
+      mailSubject = "メールアドレスの認証を行ってウィキアを始めよう！";
+    } else {
+      mailSubject = "Confirm your email and get started on Wikia!";
+    }
     String www = MailFunctions.getActivationLinkFromEmailContent(
-        MailFunctions.getFirstEmailContent(email, password, "Confirm your email and get started on Wikia!"));
+        MailFunctions.getFirstEmailContent(email, password, mailSubject));
     PageObjectLogging.log("getActivationLinkFromMail",
                           "activation link is visible in email content: " + www, true);
     return www;
   }
 
-  public ConfirmationPageObject enterActivationLink(String email, String password, String wikiURL) {
-    getUrl(getActivationLinkFromMail(email, password));
+  public ConfirmationPageObject enterActivationLink(String email, String password, String wikiURL, String language) {
+    getUrl(getActivationLinkFromMail(email, password, language));
     PageObjectLogging.log("enterActivationLink", "activation page is displayed", true, driver);
     return new ConfirmationPageObject(driver);
   }
 
+  public ConfirmationPageObject enterActivationLink(String email, String password, String wikiURL) {
+    ConfirmationPageObject confirmPage = enterActivationLink(email, password, wikiURL, "english");
+    return confirmPage;
```

Comment:
This is a readibility topic I'd accept leaving it in separate variable.
enterActivationLink doesn't indicate the method returns confirmationPage
On the other hand it's clear in the declared return class
Considering the return class perspective I agree with Tomek